hadoop-ozone/tools/pom.xml

```diff
@@ -59,6 +59,18 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>hadoop-hdfs</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
```

Comment:
This needs to be excluded from hadoop-common dependency like below:

    <dependency>
      <groupId>org.apache.hadoop</groupId>
      <artifactId>hadoop-common</artifactId>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <groupId>com.sun.xml.bind</groupId>
          <artifactId>jaxb-impl</artifactId>
        </exclusion>
      </exclusions>
    </dependency>

Reply:
Hi @xiaoyuyao, thanks for the review.  This change adds 3 dependencies, but none of them is a transitive dependency via `hadoop-common`.  Can you please clarify what needs to be excluded and why?

Reply:
Agree, since you don't add jaxb-impl. We should be good. There is no need to exclude.